packages/react-celo/src/use-celo.tsx

```diff
@@ -57,15 +58,20 @@ export function useCelo<CC = undefined>(): UseCelo {
 
   return {
     dapp,
-    address,
+    address, // The account address
     network,
     feeCurrency,
     initError: connectorInitError,
     // Copy to ensure any accidental mutations dont affect global state
     networks: networks.map((net) => ({ ...net })),
 
     kit: connector.kit,
-    account: connector.account,
+    // the wallet address from Account.getWalletAddress => The address at which the account expects to receive transfers.
```

Comment:
CONSIDER: 5.0 standarize the use of fetchWalletAddress with defaultAccount vs address